Gems/WarehouseAssets/Code/Include/WarehouseAssets/WarehouseAssetsBus.h

```diff
@@ -0,0 +1,31 @@
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Interface/Interface.h>
+
+namespace WarehouseAssets
+{
+    class WarehouseAssetsRequests
```

Comment:
question: As this is an asset only Gem (or at least appears to be) could we delete all this autogenerated code? Is it actually needed for anything? Again @lumberyard-employee-dm might be able to weigh in on this. Thanks!

Reply:
Changed to asset Gem and removed code.

Reply:
Awesome thank you!